server/game/cards/23-AHaH/TheHighroad.js

```diff
@@ -0,0 +1,36 @@
+const DrawCard = require('../../drawcard.js');
+
+class TheHighroad extends DrawCard {
+    setupCardAbilities(ability) {
+        this.plotModifiers({
+            gold: 1
+        });
+        this.action({
+            title: 'Kneel and sacrifice',
+            clickToActivate: true,
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.sacrificeSelf()
+            ],
+            handler: context => {
+                // Because the Highroad ends up in the discard pile of its owner
```

Comment:
I used the Kingsroad as an inspiration and there was this comment. I think it also applies to the Highroad hence I kept it here as well.